front/components/items/Appointment/helper.ts

```diff
@@ -28,3 +28,22 @@ export const getDateOfAppointments = ({ appointments }: props) => {
         }
     });
 };
+
+export const getIdOfAppointment = ({
+    fullDataAppointment,
+    chosenTherapist,
+    chosenDate,
+    chosenTime,
+}: propsOfGetId) => {
```

Comment:
In arrow function you can do simpler return if there is no code other that return in the function.

=> {return X} can be replaced by => X